### PR TITLE
Task s1 testing

### PR DESCRIPTION
**Pull request by C0deJack**
Tests completed for all story one: 

1. AdminControllerTest
2. LoginControllerTest
3. AdminControllerFactoryTest
4. LoginControllerFactoryTest
5. UserModelFactoryTest
6. UserModelTest

### CODE REVIEW

File: tests/Factories/AdminControllerFactoryTest.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+use PHPUnit\Framework\TestCase;
+use Portal\Factories\AdminControllerFactory;
+use Slim\Views\PhpRenderer;
+
+class AdminControllerFactoryTest extends TestCase
+{
+    function testInvoke()
+    {
+        $stub = $this->createMock(PhpRenderer::class);
```
Comment (karimkefi) [inline tests/Factories/AdminControllerFactoryTest.php]: similar to the tests/Factories/LoginControllerFactoryTest.php, you should be replicating the real factory:
$container = $this->createMock(ContainerInterface::class);
then
$container->method('get')->willReturn...

File: tests/Factories/UserModelFactoryTest.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+use PHPUnit\Framework\TestCase;
+use Portal\Factories\UserModelFactory;
+use Portal\Models\UserModel;
+use Psr\Container\ContainerInterface;
+
+class UserModelFactoryTest extends TestCase
+{
+    function testInvoke()
+    {
+        $db = new PDO('mysql:host=192.168.20.20;dbname=academyPortal', 'root');
```
Comment (karimkefi) [inline tests/Factories/UserModelFactoryTest.php]: Dont create a new PDO everytime. you need to mock the PDO

File: tests/Models/UserModelTest.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use Portal\Models\UserModel;
+
+class UserModelTest extends TestCase
+{
+    function testConstruct()
+    {
+            $db = new PDO('mysql:host=192.168.20.20;dbname=academyPortal', 'root');
```
Comment (karimkefi) [inline tests/Models/UserModelTest.php]: as above, mock the PDO (dont created a new connection)

File: tests/Factories/AdminControllerFactoryTest.php (hunk 1 above)
Comment (C0deJack) [reply tests/Factories/AdminControllerFactoryTest.php]: done

File: tests/Models/UserModelTest.php (hunk 3 above)
Comment (C0deJack) [reply tests/Models/UserModelTest.php]: as above

**Pull request merged**